### DISCUSSION TITLE
Temperature Card Option

### DISCUSSION THREAD

**Discussion by Jamesfb09** [Rem0o/FanControl.Releases#527] [Ideas]
Is it possible to have an option to allow for a "Temperature Card" instead of the tray icon? I don't use up all of the space for Fan Control Cards and RPM cards. An option to add temperature cards for the different temp sensors would be cool or an option to display the temps with the corresponding sensors on the existing cards, like under the RPM display or something. 

Either way, this program is amazing and I love using it!

**Comment by Rem0o** [comment]
Well, the temperature is already shown on the fan curve cards.    FanControl is not made to monitor stuff. It only shows what's being used. 

Like so:
![image](https://user-images.githubusercontent.com/14118956/157767503-f5f424fc-c90c-42a6-b71e-223fb27a7bee.png)

**Comment by Jamesfb09** [comment]
Ah okay! That's honestly a section I haven't looked at yet. I think I can use this for doing exactly what I was asking for. Thank you for the insanely quick response! Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Rem0o ***@***.***> Date: 3/10/22  5:44 PM  (GMT-05:00) To: "Rem0o/FanControl.Releases" ***@***.***> Cc: Jamesfb09 ***@***.***>, Author ***@***.***> Subject: Re: [Rem0o/FanControl.Releases] Temperature Card Option (Discussion #527) 
Well, the temperature is already shown on the fan curve cards.    FanControl is not made to monitor stuff. It only shows what's being used.
Like so:

**Comment by Jamesfb09** [comment]
After taking a further look it appears I can't display the fan curve graphs
or even find anywhere to set the graphs. When I click on curve instead of
manual, the only thing I can do is change the percentages, Is this due to
an incompatibility with my hardware? I think this is why I never used the
curve setting in the first place, cause I could never see the graphs or do
anything with the curves.

My Mobo is Gigabyte and the bios has Fan curves setup, but I just override
them manually with your program as needed between gaming and idle. I'm not
using Fan Control to specifically monitor anything I really just wanted to
be able to see exactly what you showed me the program should be able to
display when it comes to temps and the sensors.

Ultimately if this is how it will work for me that's totally fine, the
program does exactly what I need it to and keeps my temps under control. It
was more of a want then a necessity. Which speaking of, I was also
wondering if there is a possibility to have the ability to hot-key the
profiles for easy switching?

Thanks again for your reply! I appreciate the feedback.
[image: c7fe995f-4b02-46d3-a409-e7cdcc1c885a.png]

**Reply by Rem0o** [reply]
You should be using the curves, that's the whole point of the software. Manual mode is just a debugging tool mostly.

Click the big + button at the bottom right to create a curve. Select "Graph" in the selection.
A card should be created.   Assign a temp sensor to it, edit the curve, then name the card something meaningful.
Then go back to whatever control card you want, untick manual mode, and select your newly created curve from the "curve" drop down of that card. The control will now FOLLOW that graph.

**Comment by Jamesfb09** [comment]
Oh lol. I feel a bit silly. I see that plus sign all the time and not even
once did I think to click it...Doh!! Thank you so much for your help! I
really appreciate it. I'm going to get the curves all set up today and see
how it goes! If I have anymore questions I'll let you know. Have a great
day! Thanks again!

**Comment by Brigadir2** [comment]
Unfortunately, the video card does not work. nVIDIA GeForce RTX 3080 Ti Founders Edition
![Fan Cntrol](https://user-images.githubusercontent.com/113856756/190924264-87878fd7-007c-4da7-99c8-79f956699e5b.jpg)
